kotlin/viewshed-location/README.md

```diff
@@ -0,0 +1,34 @@
+# Viewshed location
+
+Perform a viewshed analysis from a defined vantage point.
+
+![Image of viewshed location](viewshed-location.png)
+
+## Use case
+
+A 3D viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed shows what can be seen from a given location. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red). Viewshed analysis is a form of "exploratory analysis", which means the results are calculated on the current scale of the data, and the results are generated very quickly. If more "conclusive" results are required, consider using a `GeoprocessingTask` to perform a viewshed instead.
+
+## How to use the sample
+
+Use the sliders to change the properties (heading, pitch, etc.), of the viewshed and see them updated in real time. To move the viewshed, double touch and drag your finger across the screen. Lift your finger to stop moving the viewshed.
+
+## How it works
+
+1. Create a `LocationViewshed` passing in the observer location, heading, pitch, horizontal/vertical angles, and min/max distances.
+2. Set the property values on the viewshed instance for location, direction, range, and visibility properties.
+
+## Relevant API
+
+* AnalysisOverlay
+* ArcGISTiledElevationSource
+* ArcGISSceneLayer
+* LocationViewshed
+* Viewshed
+
+## About the data
+
+The scene in this sample shows a [buildings layer](https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0) in Brest, France.
+
+## Tags
+
+visibility analysis, viewshed, LocationViewshed, AnalysisOverlay, Scene, 3D, frustum
```

Comment:
looks like this is another example from the common-sample readme which contains references to the API in the tags: let's remove LocationViewshed and AnalysisOverlay since they are already in the Relevant API section. I think it's ok to leave Scene in here though (it's not the focus of the sample, so it doesn't get put in the Relevant API section, but folk who search for "Scene" could still find this).